gnomad_qc/v4/sample_qc/relatedness.py

```diff
@@ -176,26 +341,78 @@ def main(args):
         default=4,
         type=int,
     )
-    parser.add_argument(
-        "--min-kin-cutoff",
-        help=(
-            "Minimum kinship threshold for a pair of samples to be retained in the "
-            "cuKING output."
-        ),
-        default=0.05,
-        type=float,
-    )
-    parser.add_argument(
+    cuking_args.add_argument(
         "--create-cuking-relatedness-table",
         help="Convert the cuKING outputs to a standard Hail Table.",
         action="store_true",
     )
-    parser.add_argument(
-        "--relatedness-n-partitions",
-        help="Number of desired partitions for the relatedness Table.",
-        default=50,
+    print_cuking_ibd = cuking_args.add_argument_group(
```

Comment:
is the "print" part here necessary? Looks like the args in this group are for actually running steps not just printing?